packages/babel-traverse/test/scope.js

```diff
@@ -60,5 +73,13 @@ describe("scope", function () {
         _foo2: { }
       `).scope.generateUid("foo"), "_foo3");
     });
+
+    it("reference paths", function() {
+      const path = getIdentifierPath("function square(n) { return n * n}");
+      const referencePaths = path.context.scope.bindings.n.referencePaths;
```

Comment:
unrelated: I guess we should have some tests for `scope.bindings` (check that only n is defined as a binding)